src/tests/dynamic_imports_test/CodeSplittingFileIntegrity.test.ts

```diff
@@ -705,6 +706,58 @@ export class CodeSplittingFileIntegrityTest {
             }));
     }
 
+    "Should if check browser data is it correctly"() {
+        return FuseTestEnv.create(
+            {
+                project: {
+                    files: {
+                        "index.ts": `
+                            export async function getRemoteFile() {
+                                await import("./components/HomeComponent")
+                            }
+                        `,
+                        "components/HomeComponent.ts": `
+                            export function home(){ return "home"  }
+                        `,
+
+                    },
+                    plugins: [
+                        QuantumPlugin({
+                            target: "browser",
+                            runtimeBundleMapping: "newBundleMapping"
+                        })
+                    ]
+                }
+            }
+        )
+            .simple().then(test => test.browser((window, env) => {
+                window['newBundleMapping'] = {
+                    c: {
+                        b: './asdasd',
```

Comment:
Let's give it real values, and see if it's possible to import without errors

Reply:
Well it doesn't fail with these 'non-real' values.
I wanted to make it fail first and then correct it with real values.

Reply:
It should fail on 
````
window.$fsx.r(0);
```
right?
Or am i missing the point

Reply:
can you hack into `FuseTestEnv` and add this `preloadScript` as an optional argument? That should do the trick

Reply:
So i successfully extended FuseTestEnv and added the preloading script.
I'm having issues validating, that the dynamic loading works properly.

```
 .simple().then(test => test.browser((window, env) => {
                var index = window.$fsx.r(0);
                index.getRemoteFile().then(result => {
                    should(result).equal("home");
                });
            }, bundleMappingScript));
```

Seems like async operations aren't possible within the first argument of FuseTestEnv.browser?

Reply:
How is it going with it? Do you need help? You can just return it
```
 var index = window.$fsx.r(0);
                return index.getRemoteFile().then(result => {
                    should(result).equal("home");
                });
```

I can help you with this. Let's merge it already ;-)

Reply:
Hey I am really busy this week, I'll see what i can do. I might not be able to check in before friday.